### PR TITLE
check parl and python version

### PR DESCRIPTION
**Pull request by YuechengLiu**
match the parl & python version running on  'master', 'client' and 'worker'

### CODE REVIEW

File: parl/remote/worker.py [hunk 1]
```diff
@@ -101,6 +102,23 @@ def _set_cpu_num(self, cpu_num=None):
         else:
             self.cpu_num = multiprocessing.cpu_count()
 
+    def check_version(self):
```
Comment (TomorrowIsAnOtherDay) [inline parl/remote/worker.py:105]: Please add comment for this function.

File: parl/remote/worker.py [hunk 2]
```diff
@@ -101,6 +102,23 @@ def _set_cpu_num(self, cpu_num=None):
...
+        self.request_master_socket.send_multipart(
+            [remote_constants.CHECK_VERSION_TAG])
+        message = self.request_master_socket.recv_multipart()
+        tag = message[0]
+        if tag == remote_constants.NORMAL_TAG:
+            worker_parl_version = parl.__version__
+            worker_python_version = str(sys.version_info.major)
+            assert worker_parl_version == to_str(message[1]) and worker_python_version == to_str(message[2]),\
+                '''Version mismatch: the "master" is of version "parl={}, python={}",however,
```
Comment (TomorrowIsAnOtherDay) [inline parl/remote/worker.py]: However, XXX is provided in your environment.

File: parl/remote/client.py [hunk 3]
```diff
@@ -162,6 +164,23 @@ def _create_sockets(self, master_address):
...
+    def check_version(self):
+        self.submit_job_socket.send_multipart(
+            [remote_constants.CHECK_VERSION_TAG])
+        message = self.submit_job_socket.recv_multipart()
+        tag = message[0]
+        if tag == remote_constants.NORMAL_TAG:
+            client_parl_version = parl.__version__
+            client_python_version = str(sys.version_info.major)
+            assert client_parl_version == to_str(message[1]) and client_python_version == to_str(message[2]),\
+                '''Version mismatch: the 'master' is of version 'parl={}, python={}',however,
```
Comment (TomorrowIsAnOtherDay) [inline parl/remote/client.py]: However, XX is provided in your environment.

File: parl/remote/client.py [hunk 4]
```diff
@@ -162,6 +164,23 @@ def _create_sockets(self, master_address):
                             "check if master is started and ensure the input "
                             "address {} is correct.".format(master_address))
 
+    def check_version(self):
```
Comment (TomorrowIsAnOtherDay) [inline parl/remote/client.py:167]: Please add comment for this function.

File: parl/remote/worker.py (hunk 2 above)
Comment (YuechengLiu) [reply parl/remote/worker.py]: modified

**Review by rlagent333 — APPROVED** [review]

**Review by TomorrowIsAnOtherDay — APPROVED** [review]

**Pull request merged**